app/javascript/dashboard/components/layout/sidebarComponents/AddAccountModal.vue

```diff
@@ -90,3 +100,25 @@ export default {
   },
 };
 </script>
+<style lang="scss">
+.activity-wrap {
```

Comment:
Can we reuse callouts to avoid styling ? => https://get.foundation/sites/docs/callout.html

Reply:
This makes sense to me.